apps/explorer/src/api/assets/assetsApi.ts

```diff
@@ -167,7 +167,15 @@ export const assetsQueries = {
         queryFn: async (): Promise<NFTTokenUriMetaData & { assetId: string }> => {
           const nftData = (await axios.get(dataUri)).data as NFTTokenUriMetaData
 
-          return matchesNFTTokenUriMetaDataSchema(nftData) ? { ...nftData, assetId } : Promise.reject()
+          return matchesNFTTokenUriMetaDataSchema(nftData)
+            ? { ...nftData, assetId }
+            : !!nftData?.name || !!nftData?.image
+              ? {
+                  assetId,
+                  name: nftData?.name ? nftData?.name.toString() : 'Unsupported NFT',
```

Comment:
Shouldn't `Unsupported NFT` be translated, like `Unsupported media type` is?

Reply:
Honestly, if an NFT doesn't even have a name, I would reject the promise. If it doesn't have an image though we can still show its name. WDYT?

Reply:
Yeah I agree! 👍

Reply:
Fixed